Tests/Invoke-Jenkins.Tests.ps1

```diff
@@ -1,4 +1,4 @@
- Import-Module -Name $PSScriptRoot\..\ThreeShape.Jenkins.Beta.psm1
+ Import-Module -Name (Get-Item $PSScriptRoot\..\*.psm1)[0].FullName
```

Comment:
```suggestion
Import-Module -Name (Get-ChildItem $PSScriptRoot\..\*.psm1 | Select-Object -first 1).FullName
```

or

```suggestion
$module = (Get-ChildItem $PSScriptRoot\..\*.psm1 | Select-Object -first 1)
Import-Module -Name $module.FullName
```

Reply:
I don't believe option 2 will compile, actually. But the first suggestion is more readable and a definite improvement.

Reply:
Done.